Operators/Types/user/community/SnapToMultiple.cs

```diff
@@ -0,0 +1,38 @@
+using T3.Core.Operator;
+using T3.Core.Operator.Attributes;
+using T3.Core.Operator.Slots;
+
+namespace T3.Operators.Types.Id_c3090c65_194a_4f95_9b70_d003f54103f7 
+{
+    public class SnapToMultiple : Instance<SnapToMultiple>
+    {
+        [Output(Guid = "87a311ce-238c-472c-b43e-e4ed5268bbc5")]
+        public readonly Slot<int> Result = new();
+
+        public SnapToMultiple()
+        {
+            Result.UpdateAction = Update;
+        }
+
+        private void Update(EvaluationContext context)
+        {
+            var v = Value.GetValue(context);
+            var mod = Mod.GetValue(context);
+            if (mod == 0){
```

Comment:
this part is not needed, in this case the "end test" (second field) of the `for` statement will trigger before the first run, and the loop content won't be executed.